lib/trento/domain/host/host.ex

```diff
@@ -1,5 +1,20 @@
 defmodule Trento.Domain.Host do
-  @moduledoc false
+  @moduledoc """
+  The host aggregate manages all the domain logic related to individual hosts
+  (agent, in other term) that compose the target infrastructure running SAP systems.
+
+  Each host running a Trento agent is registered as a new aggregate entry. The host
+  aggregate stores information such as:
+
+  - Host basic information as the hostname and ip addresses
+  - Hardware specifications
+  - Platform where the host is running (the cloud provider for instance)
+  - Registered SLES4SAP subscriptions
+
+  Besides these mostly static values, the aggregate takes care of detecting the host availability
```

Comment:
I would make also a reference to the fact that the aggregate detects changed discovered information and updates the system.

Not a biggie tho.